lib/ping_pong/events/life_stop.js

```diff
@@ -12,9 +12,11 @@
 const onLifeStop = async (instance = {}) => {
   const { state = {}, h = {} } = instance
   const { args = {}, orders = {}, gid } = state
-  const { emit } = h
+  const { emit, debug } = h
   const { cancelDelay } = args
 
+  debug('detected ping/pong algo cancelation, stopping...')
+
   return emit('exec:order:cancel:all', gid, orders, cancelDelay)
```

Comment:
should be set to 0 here, else we have ghost orders after the user clicks cancel

Reply:
ok!! will change to 0